rust/userborn/src/main.rs

```diff
@@ -440,7 +438,7 @@ mod tests {
         let expected_group = expect![[r#"
             root:x:0:root
             initial:x:998:initial
-            wheel:x:999:normalo,initial
+            wheel:x:999:initial,normalo
```

Comment:
Those has to be sorted now. But I have to say having reproducible order in this file is nicer anyway.